OMCompiler/Compiler/NFFrontEnd/NFInstNode.mo

```diff
@@ -31,19 +31,25 @@
 
 encapsulated package NFInstNode
 
+import Binding = NFBinding;
 import Component = NFComponent;
 import Class = NFClass;
 import SCode;
 import Absyn;
 import AbsynUtil;
 import Type = NFType;
+import Typing = NFTyping;
+import Inst = NFInst;
+import InstContext = NFInstContext;
 import NFFunction.Function;
 import Sections = NFSections;
 import Pointer;
 import Error;
+import Flatten = NFFlatten;
+import NFFlatten.Prefix;
 import Prefixes = NFPrefixes;
 import Visibility = NFPrefixes.Visibility;
-import NFModifier.Modifier;
+import NFModifier.{Modifier, ModifierScope};
```

Comment:
None of the added imports seem to be used, leftovers from some previous attempt perhaps? Same in `NFVariable.mo` too.